boards/ferris_sweep/kb.py

```diff
@@ -0,0 +1,48 @@
+# Ferris Sweep pinout translated from QMK repo
+# CreditT: 2018-2020 ENDO Katsuhiro, David Philip Barr, Pierre Chevalier
+
+import board
+
+from kmk.kmk_keyboard import KMKKeyboard as _KMKKeyboard
+from kmk.quickpin.pro_micro.kb2040 import pinout as pins
+
+from kmk.scanners.keypad import KeysScanner
+
+from storage import getmount
+
+# GPIO to key mapping - each line is a new row.
+# fmt: off
+_KEY_CFG_LEFT = [
+     pins[9], pins[16], pins[17], pins[18], pins[19],
+    pins[15], pins[14], pins[13], pins[12],  pins[0],
+     pins[4],  pins[5],  pins[6],  pins[7],  pins[8],
+                                  pins[10], pins[11],
+]
+
+_KEY_CFG_RIGHT = [
+    pins[19], pins[18], pins[17], pins[16],  pins[9],
+     pins[0], pins[12], pins[13], pins[14], pins[15],
+     pins[8],  pins[7],  pins[6],  pins[5],  pins[4],
+    pins[11], pins[10],                        
+]
```

Comment:
With the `coord_mapping` there's no reason to have left and right side pin definitions. They are identical, only mirrored.
Try reversing the rows of your right side, for example:
```python
coord_mapping = [
0, 1, 2, 3, 4,   21, 20, 19, 18, 17,
...]
```

Reply:
That's a much better way of doing it! Thanks